docs/Contribute.md

```diff
@@ -0,0 +1,32 @@
+# Contribution Guidelines
+
+Thanks for taking the time to contribute!
+
+The following is a set of guidelines for contributing to SecDevLabs. These are just guidelines, not rules, so use your best judgement and feel free to propose changes to this document in a pull request! 😃
+
+## So... I built this app, it looks great, but how can I send it to SecDevLabs?
+
+Well, a good place to start is to make sure you can deploy your app with Docker-Compose. All the apps in SecDevLabs are built with it in order to ensure you can get get right into the action! ✓
+
+So your app can be deployed with Docker-Compose, nice job! But wait, does it have a Makefile? If it doesn't, be sure to have a look [here][1], we have a template you can follow to create your own! ✓
```

Comment:
Suggestion: `we have a template you may follow or you can create your own`

Reply:
Thanks @rafaveira3 ! Liked your suggestion!